packages/gestalt/src/Row.js

```diff
@@ -39,37 +42,23 @@ type Props = {|
   wrap?: boolean,
 |};
 
-export default function Row({
-  alignItems = 'center',
-  children,
-  gap = 0,
-  height,
-  justifyContent = 'start',
-  width,
-  ...rest
-}: Props): Node {
+// Defaults for align-items and justify-content set in base styles
+export default function Row({ children, gap = 0, ...rest }: Props): Node {
+  const baseClasses = classnames(
+    styles.row,
+    styles[`horizontalGap${gap}`],
+    styles[`horizontalGap${gap} > *`]
```

Comment:
using the child combinator to prevent needing to add another div. this allows children to set styles like `flex="grow"`